portfolio/src/main/webapp/index.html

```diff
@@ -48,8 +33,7 @@
     ></script>
     <script src="script.js"></script>
   </head>
-
-  <body>
+  <body class = "bd" onload="createMap();">
```

Comment:
[Style](https://google.github.io/styleguide/htmlcssguide.html#HTML_Validity): Use valid HTML where possible. 

Try copying/pasting this file on https://validator.w3.org/nu/#textarea

Reply:
Running `prettier` on the file will also show the same errors